recipe/meta.yaml

```diff
@@ -1,6 +1,6 @@
 {% set name = "rubin-env" %}
-{% set version = "5.0.0" %}
-{% set build_number = 2 %}
+{% set version = "5.0.1" %}
```

Comment:
In theory, a `<` pin can be a new build, but you need the patch to avoid reusing already-built environments?

Reply:
Not increasing the build number makes sense. Old environments won't build otherwise. Fixing it.